languages/elixir/exercises/concept/anonymous-functions/.meta/example.ex

```diff
@@ -0,0 +1,25 @@
+defmodule ClosureMaker do
```

Comment:
Looking at this example, I feel that this is exercise looks to be relatively, well, dull? Each function has almost exactly the same format: returning a one parameter anonymous function. Maybe you could mix things up a bit, and e.g. return functions with more than one parameter or possible require calling a library function that takes an anonymous function as one of its arguments (this would allow students to use the shorthand notation).

Another issue might be that there is not a real story/theme to work with. Maybe you could try to come up with a theme?

Reply:
I have added a theme to this.  Take a peek at the revision?